openapi/components/requestBodies/Webhooks/InvoiceRevenueRecognized.yaml

```
@@ -3,16 +3,16 @@ content:
     schema:
       properties:
         recognitionId:
-          description: The invoice revenue recognition ID.
+          description: Revenue recognition ID of the invoice.
```

```suggestion
          description: ID of the revenue recognition.
```

---

Are losing information here on the invoice aspect?

---

```suggestion
          description: ID of the invoice revenue recognition.
```

Maybe this would be better.

---

Updated to: `ID of the recognized revenue.`

 ([2abb663](https://github.com/Rebilly/api-definitions/pull/1044/commits/2abb6634ffc00945798ff1331669f64df1566dd1))